server/routes/messages.js

```diff
@@ -13,17 +13,13 @@ const pusher = new Pusher({
   secret: IAM_SECRET,
 });
 
-let messages = [];
-
-router.get('/messages', (req, res) => {
-  res.send('all good');
-});
 
 router.post('/pusher/auth', (req, res) => {
```

Comment:
One thing to double check with Shad is that now that the routes for pusher are on `/api/messages/pusher/auth` is everything still working on the client side?